resources/migrations/000_migrations.yaml

```diff
@@ -9881,6 +9881,15 @@ databaseChangeLog:
                   type: ${text.type}
                   remarks: "Serialized JSON containing Database-local Settings for this Database"
 
+  - changeSet:
+      id: v42.00-086
+      author: noahmoss
+      comment: >-
+        Added 0.42.0 - migrates the Sample Dataset to the name "Sample Database"
+      changes:
+        - sql:
+            sql: UPDATE metabase_database SET name='Sample Database' WHERE is_sample=true
+
```

Comment:
Here's the new migration, for reference

Reply:
I had claimed a bunch of migration numbers for pulses but we are waiting on those. This migration should be renumbered to `v42.00-071`. That's my fault and I apologize for not unclaiming those earlier.